src/docs/torque.md

```diff
@@ -49,7 +49,8 @@ Hello world!
 
 The Torque compiler doesn’t create machine code directly, but rather generates C++ code that calls V8’s existing `CodeStubAssembler` interface. The `CodeStubAssembler` uses the [TurboFan compiler’s](https://v8.dev/docs/turbofan) backend to generate efficient code. Torque compilation therefore requires multiple steps:
 
-1. The `gn` build first runs the Torque compiler. It processes all `*.tq` files, outputting corresponding `*-gen.cc` files (one `.cc` file per Torque namespace). The `.cc` files that are generated use TurboFan’s `CodeStubAssembler` interface for generating code.
+1. The `gn` build first runs the Torque compiler. It processes all `*.tq` files, outputting corresponding `*-gen.cc` files (one `.cc` file per Torque namespace). The `.cc` files that are generated use TurboFan’s `CodeStubAssembler` interface for generating code. The Torque compiler also generates various `.h` files meant to be consumed by the V8 build for any class definitions found in `.tq` files.
```

Comment:
"one `.cc` file per Torque namespace" is no longer true, it's one file per Torque file now. 
path/to/file.tq turns into gen/torque-generated/path/to/file-tq-csa.{h,cc}.